mmhuman3d/models/body_models/smplx.py

```diff
@@ -371,5 +392,330 @@ def forward(self,
             output['vertices'] = smplx_output.vertices
         if return_full_pose:
             output['full_pose'] = smplx_output.full_pose
+        return output
+
 
+@dataclass
+class ModelOutput(SMPLXOutput):
+    """The name of the model output."""
+    smpl_joints: Optional[torch.Tensor] = None
+    joints_J19: Optional[torch.Tensor] = None
+    smplx_vertices: Optional[torch.Tensor] = None
+    flame_vertices: Optional[torch.Tensor] = None
+    lhand_vertices: Optional[torch.Tensor] = None
+    rhand_vertices: Optional[torch.Tensor] = None
+    lhand_joints: Optional[torch.Tensor] = None
+    rhand_joints: Optional[torch.Tensor] = None
+    face_joints: Optional[torch.Tensor] = None
+    lfoot_joints: Optional[torch.Tensor] = None
+    rfoot_joints: Optional[torch.Tensor] = None
+
+
+class SMPLX_ALL(nn.Module):
+    """Extension of the official SMPLX implementation to support more
+    joints."""
+    SMPLX2SMPL_J45 = [i for i in range(22)
+                      ] + [30, 45] + [i for i in range(55, 55 + 21)]
+    JOINT_MAP = {
```

Comment:
Shall we merge the `JOINT_MAP` into [`convert_kps`](https://github.com/open-mmlab/mmhuman3d/blob/main/mmhuman3d/core/conventions/keypoints_mapping/__init__.py#L83)? It has a [get_mapping](https://github.com/open-mmlab/mmhuman3d/blob/main/mmhuman3d/core/conventions/keypoints_mapping/__init__.py#L234) function to get the mapping via matching joint names.